src/lock_file/resolve/pypi.rs

```diff
@@ -401,8 +405,25 @@ pub async fn resolve_pypi(
     .into_diagnostic()
     .context("failed to resolve pypi dependencies")?;
 
+    if package_requests.borrow().len() > 0 {
+        // Print package requests in form of (PackageName, NumRequest)
+        let package_requests = package_requests
+            .borrow()
+            .iter()
+            .map(|(name, value)| format!("[{name}: {value}]"))
+            .collect::<Vec<_>>()
+            .join(",");
+        tracing::info!("overridden uv PyPI package requests [name: amount]: {package_requests}");
```

Comment:
Should these perhaps be debug statements?

Reply:
Perhaps, indeed. We should maybe figure out a better way than just the default verbosity flags as debug is very noisy currently. Would be good to filter out any debugs from underlying libraries.